Description: Do not force the installation of the Dorkbox tray (should be packaged separately)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/uis/src/com/biglybt/ui/swt/systray/SystemTraySWT.java
+++ b/uis/src/com/biglybt/ui/swt/systray/SystemTraySWT.java
@@ -124,7 +124,7 @@
 				SystemTraySWT.core = core;
 				gm = core.getGlobalManager();
 				if (Constants.isUnix) {
-					ensureDorkBoxPlugin(core);
+					//ensureDorkBoxPlugin(core);
 				}
 			}
 		});
